webpack.config.cli.dev.js

```diff
@@ -53,28 +53,31 @@ devConfig.module.rules.push({
     {
       loader: 'css-loader',
       options: {
-        localIdentName: '[local]---[hash:base64:5]',
-        modules: true,
+        modules: {
+          localIdentName: '[local]---[hash:base64:5]',
+        },
         sourceMap: true,
         importLoaders: 1,
       },
     },
     {
       loader: 'postcss-loader',
       options: {
-        plugins: () => [
-          postCssImport(),
-          autoprefixer(),
-          postCssCustomProperties({
-            preserve: false,
-            importFrom: [locateCssVariables()]
-          }),
-          postCssCalc(),
-          postCssNesting(),
-          postCssCustomMedia(),
-          postCssMediaMinMax(),
-          postCssColorFunction(),
-        ],
+        postcssOptions: {
+          plugins: [
+            postCssImport(),
+            autoprefixer(),
+            postCssCustomProperties({
+              preserve: false,
+              importFrom: './lib/variables.css'
```

Comment:
On #master, `lib/variables.css` was replaced with `[locateCssVariables()]` in #4. Did you undo that intentionally?

Reply:
I definitely did not intend this... committing the correction...